docs/getting-started/supported-languages-and-tools.md

```diff
@@ -197,6 +218,7 @@ The table below lists all languages and frameworks that Codacy supports and the
           <a href="https://pmd.github.io/">PMD</a></td>
       <td><a href="https://eslint.org/docs/rules/">ESLint</a> <a href="#suggest-fixes">🔧</a></td>
       <td><a href="https://trivy.dev">Trivy</a></td>
+      <td><a href="https://trivy.dev">Trivy</a> (scans <br><code>package-lock.json</code>, <br><code>yarn.lock</code>) <a href="#any-path"><sup>3</sup></a></td>
```

Comment:
are these enough, or would it be also required to add the "package.json"? 🤔

Reply:
I'll mention the tool name as well. I'd keep the file name since that's what's actually scanned; here I'm assuming that it may be useful for the user to have a written confirmation that a specific file is/isn't being scanned.